meta-zarhus-distro/recipes-tools/zarhus-memtools/zarhus-memtools/heap_analyze

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Is this script needed? It does less than nothing:

* It forwards first argument to heaptrack as individual arguments split by space except -h/--help argument
* --help doesn't print anything useful

just use `heaptrack` directly e.g. `heaptrack cog <url>`.

Reply:
Deleted e2f1787aaf4d9a17dcef5ebc771477c204d70aa8